Does not build on Windows

```
   Compiling renderdoc v0.6.0
error[E0412]: cannot find type `ComPtr` in this scope
  --> C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\renderdoc-0.6.0\src\handles.rs:44:11
   |
44 | impl From<ComPtr<winapi::um::d3d11::ID3D11Device>> for DevicePointer {
   |           ^^^^^^ not found in this scope
help: possible candidate is found in another module, you can import it into scope
   |
3  | use wio::com::ComPtr;
   |
error[E0412]: cannot find type `ComPtr` in this scope
  --> C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\renderdoc-0.6.0\src\handles.rs:45:18
   |
45 |     fn from(ctx: ComPtr<winapi::um::d3d11::ID3D11Device>) -> Self {
   |                  ^^^^^^ not found in this scope
help: possible candidate is found in another module, you can import it into scope
   |
3  | use wio::com::ComPtr;
   |
error[E0412]: cannot find type `ComPtr` in this scope
  --> C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\renderdoc-0.6.0\src\handles.rs:58:11
   |
58 | impl From<ComPtr<winapi::um::d3d12::ID3D12Device>> for DevicePointer {
   |           ^^^^^^ not found in this scope
help: possible candidate is found in another module, you can import it into scope
   |
3  | use wio::com::ComPtr;
   |
   Compiling wasmer-wasi v0.6.0
error[E0412]: cannot find type `ComPtr` in this scope
  --> C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\renderdoc-0.6.0\src\handles.rs:59:18
   |
59 |     fn from(ctx: ComPtr<winapi::um::d3d12::ID3D12Device>) -> Self {
   |                  ^^^^^^ not found in this scope
help: possible candidate is found in another module, you can import it into scope
   |
3  | use wio::com::ComPtr;
   |
error[E0119]: conflicting implementations of trait `std::convert::From<[type error]>` for type `handles::DevicePointer`:
  --> C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\renderdoc-0.6.0\src\handles.rs:51:1
   |
44 | impl From<ComPtr<winapi::um::d3d11::ID3D11Device>> for DevicePointer {
   | -------------------------------------------------------------------- first implementation here
...
51 | impl From<*mut winapi::um::d3d12::ID3D12Device> for DevicePointer {
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ conflicting implementation for `handles::DevicePointer`
error[E0119]: conflicting implementations of trait `std::convert::From<[type error]>` for type `handles::DevicePointer`:
  --> C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\renderdoc-0.6.0\src\handles.rs:65:1
   |
44 | impl From<ComPtr<winapi::um::d3d11::ID3D11Device>> for DevicePointer {
   | -------------------------------------------------------------------- first implementation here
...
65 | impl<'a, T: glutin::ContextCurrentState> From<&'a glutin::Context<T>> for DevicePointer {
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ conflicting implementation for `handles::DevicePointer`
```

--- a/src/handles.rs
+++ b/src/handles.rs
@@ -3,6 +3,8 @@
 use std::os::raw::c_void;
 
 use glutin::os::ContextTraitExt;
+#[cfg(windows)]
+use wio::com::ComPtr;
 
 /// Raw mutable pointer to the OS-provided window handle.
 pub type WindowHandle = *const c_void;
@@ -43,7 +45,7 @@ impl From<*mut winapi::um::d3d11::ID3D11Device> for DevicePointer {
 #[cfg(windows)]
 impl From<ComPtr<winapi::um::d3d11::ID3D11Device>> for DevicePointer {
     fn from(ctx: ComPtr<winapi::um::d3d11::ID3D11Device>) -> Self {
-        unsafe { DevicePointer(ctx.as_raw() as *mut _ as *const c_void) }
+        DevicePointer(ctx.as_raw() as *mut _ as *const c_void)
     }
 }
 
@@ -57,7 +59,7 @@ impl From<*mut winapi::um::d3d12::ID3D12Device> for DevicePointer {
 #[cfg(windows)]
 impl From<ComPtr<winapi::um::d3d12::ID3D12Device>> for DevicePointer {
     fn from(ctx: ComPtr<winapi::um::d3d12::ID3D12Device>) -> Self {
-        unsafe { DevicePointer(ctx.as_raw() as *mut _ as *const c_void) }
+        DevicePointer(ctx.as_raw() as *mut _ as *const c_void)
     }
 }
 

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -45,8 +45,6 @@ use std::os::raw::c_ulonglong;
 
 #[cfg(windows)]
 use winapi::shared::guiddef::GUID;
-#[cfg(windows)]
-use wio::com::ComPtr;
 
 mod handles;
 mod renderdoc;

